README.md

```diff
@@ -6,7 +6,7 @@
   </center>
 </h1>
 
-[![npm version](https://badge.fury.io/js/one-app-router.svg)](https://badge.fury.io/js/one-app-router)
+[![npm version](https://badge.fury.io/js/%40americanexpress%2Fone-app-router.svg)](https://badge.fury.io/js/%40americanexpress%2Fone-app-router)
```

Comment:
why encoded?

Reply:
never mind the decoded version doesn't work: https://badge.fury.io/js/@americanexpress/one-app-router.svg